src/kernel/include/captbl.h

```diff
@@ -24,7 +24,11 @@
 
 #define CAPTBL_DEPTH 2
 #define CAPTBL_INTERNSZ (sizeof(int *))
-#define CAPTBL_INTERN_ORD 9 /* log(PAGE_SIZE/(2*(CAPTBL_DEPTH-1)*CAPTBL_INTERNSZ)) */
+#if defined(__x86_64__)
```

Comment:
This is an example of something not in `plat/`, and might break arm. Sorry, I can't find the context below this to see if it does ;-(

Reply:
I believe the macro difinition here will not break the arm, since it only differs between `x86_64` and `else` conditions. The `else` contains both `i386` and `arm`, thus not changing the previous code path.

Reply:
If we add AARCH64 or RISKV64 will it work?

The `ifdef` is operating on a condition that doesn't match the logical intent of what is being checked, I think. This is often a situation in which bugs creep in.

Reply:
@gparmer 
I've added a word size definition to resolve this kind of problem.